rootfs/api/serializers.py

```diff
@@ -474,3 +475,18 @@ class Meta:
         """Metadata options for a :class:`AppSettingsSerializer`."""
         model = models.AppSettings
         fields = '__all__'
+
+    def validate_whitelist(self, data):
```

Comment:
Wonder if we should use http://www.django-rest-framework.org/api-guide/fields/#ipaddressfield like we do  here https://github.com/deis/controller/blob/master/rootfs/api/serializers.py#L441-L444

Would potentially save us from custom validations